crates/runtime/tests/integration.rs

```diff
@@ -82,7 +82,7 @@ mod utils;
 mod view;
 // MySQL is required for the rehydration tests
 mod podswatcher;
-#[cfg(feature = "mysql")]
+#[cfg(all(feature = "mysql", feature = "duckdb"))]
```

Comment:
Shouldn't it be used for either feature?
```suggestion
#[cfg(any(feature = "mysql", feature = "duckdb"))]
```

Reply:
It needs both - it uses mysql as the federated dataset and duckdb for the acceleration